### PR TITLE
InfoBar

### PR DESCRIPTION
**Pull request by iamstarkov**
First commit is to prepare CrossCircle to be used properly variant="error" InfoBar

• feat(Icon.CrossCircle): add an option to modify CrossCircle's cross color via stroke prop

### CODE REVIEW

**Comment by shmidt-i** [comment]
Need to update the snapshots

File: src/Atoms/Icon/components/CrossCircle.tsx [hunk 1]
```diff
@@ -1,18 +1,26 @@
 import React from 'react';
-import { IconBase } from '../IconBase';
-import { BaseProps } from '../IconBase.types';
+import styled from 'styled-components';
+import { IconBase, getColor } from '../IconBase';
+import { ChildProps, StyledChildProps } from '../IconBase.types';
 
-export const CrossCircle = (props: BaseProps) => {
+const StyledPolygon = styled.polygon<StyledChildProps>`
```
Comment (shmidt-i) [inline src/Atoms/Icon/components/CrossCircle.tsx]: better
```
const StyledPolygon = styled.polygon<StyledChildProps>`
fill: ${getColor};
`
```
where `getColor = props => ...`


With this approach you will utilize styled-componets' inner kitchen in terms of strings interpolation, e.g. undefined could be handled nicely

Comment (iamstarkov) [reply src/Atoms/Icon/components/CrossCircle.tsx]: not sure I follow, `getColor` doesnt accept props and wasn't written by me. I stole it from [`<Icon.Star />`](https://github.com/nordnet/ui/blob/master/src/Atoms/Icon/components/Star.tsx#L6-L11)

**Comment by iamstarkov** [comment]
snapshots updated

File: src/Molecules/InfoBar/InfoBar.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    case 'warning':
+      return theme.color.text;
+    case 'general':
+      return theme.color.cta;
+    default:
+      return theme.color.cta;
+  }
+};
+
+const InfoBarPageWrapper = styled(PageWrapper)`
```
Comment (iamstarkov) [inline src/Molecules/InfoBar/InfoBar.tsx]: for some reason this doesnt work. @shmidt-i any ideas?

Comment (iamstarkov) [reply src/Molecules/InfoBar/InfoBar.tsx]: it worked in the previous pull-request

**Pull request merged**